### PR TITLE
Document --gzip option for db:dump

### PR DESCRIPTION
**Pull request by pjcdawkins**
Also bumps it above the --relationship option (as I believe it'd be more commonly wanted)

### CODE REVIEW

File: src/configuration/services/mysql.md [hunk 1]
```diff
@@ -98,17 +98,17 @@ The easiest way to download all data in a MariaDB instance is with the Platform.
 platform db:dump
 ```
 
-If you have multiple SQL databases it will prompt you which one to export. You can also specify one by relationship name explicitly:
+By default the file will be uncompressed. If you want to compress it, use the `--gzip` (`-z`) option:
 
 ```bash
-platform db:dump --relationship database
+platform db:dump --gzip
```
Comment (Crell) [inline src/configuration/services/mysql.md]: Do we only support gzip?  bzip2 (as the previous example used) has much better compression for things like giant text files.

Comment (pjcdawkins) [inline src/configuration/services/mysql.md]: Good point... for right now, gzip, but bz2 is available in the containers so I could add support for that (very soon)

Comment (Crell) [inline src/configuration/services/mysql.md]: OK. Do you want to merge these docs now and adjust later or wait for adding bzip2?  I'm fine either way.

**Comment by Crell** [comment]
I also wonder about removing the examples that include --stdout.  That's not obvious elsewhere in the docs that it exists, but it is of course useful.

**Comment by pjcdawkins** [comment]
OK, maybe we could document `--gzip` and leave in `--stdout | bzip > dump.sql.bz2`, then it shows both.

**Pull request merged**